packages/panzer/mini-em/example/BlockPrec/MiniEM_helpers.cpp

```diff
@@ -166,6 +166,10 @@ namespace mini_em {
 #ifdef KOKKOS_ENABLE_CUDA
             if (typeid(panzer::TpetraNodeType).name() == typeid(Tpetra::KokkosCompat::KokkosCudaWrapperNode).name())
               updateParams("solverMueLuRefMaxwellCuda.xml", lin_solver_pl, comm, out);
+#endif
+#ifdef KOKKOS_ENABLE_HIP
+            if (typeid(panzer::TpetraNodeType).name() == typeid(Tpetra::KokkosCompat::KokkosHIPWrapperNode).name())
+              updateParams("solverMueLuRefMaxwellCuda.xml", lin_solver_pl, comm, out);
```

Comment:
Is `solverMueLuRefMaxwellCuda.xml` the base parameter set?  The name threw me because of the `ifdef` guard.

Reply:
For the time being we can use the Cuda input deck for HIP as well. That's still better than using the CPU settings for rebalancing.